util/common/src/test/java/org/hibernate/search/util/common/impl/ToStringTreeBuilderTest.java

```diff
@@ -142,8 +142,8 @@ public String toString() {
 					.startObject( "emptyChild" )
 						.endObject()
 					.attribute( "appendable", new Appendable() )
-					.attribute( "appendableAsObject", (Object) new Appendable() )
-					.attribute( "nullAppendable", (Appendable) null )
```

Comment:
Code style: there should be a space after an opening parenthesis and before a closing parenthesis.